stable/aws-loadbalancer-controller/README.md

```diff
@@ -0,0 +1,64 @@
+# AWS Loadbalancer Controller
+
+AWS Loadbalancer controller Helm chart for Kubernetes
+
+## TL;DR:
+```sh
+helm repo add eks https://aws.github.io/eks-charts
+kubectl apply -k github.com/aws/eks-charts/stable/aws-loadbalancer-controller//crds?ref=master
+helm install eks/aws-loadbalancer-controller
+```
+
+## Introduction
+AWS Loadbalancer controller manages the following AWS resources
+- Application Load Balancers to satisfy Kubernetes ingress objects
+- Network Load Balancers in IP mode to satisfy Kubernetes service objects of type LoadBalancer with NLB IP mode annotation
+
+## Prerequisites
+- Kubernetes 1.9+ for ALB, 1.20+ for NLB IP mode
+- IAM permissions
+
+## Installing the Chart
+**Note**: You need to uninstall aws-alb-ingress-controller
```

Comment:
Does this break if aws-alb-ingress-controller is not uninstalled? Can we please elaborate a bit more and how to safe guard against conflicts. It's worth adding a section on this in the README

Reply:
Updated in the new commit.